gaussian_blur_size: 7
gaussian_blur_sigma: 2
canny_edge_th: 50
hough_accum_resolution: 1
min_circle_dist: 40
hough_accum_th: 24.7
min_radius: 20
max_radius: 150
filter_h_min: 359
filter_h_max: 10
filter_s_min: 140
filter_s_max: 255
filter_v_min: 5
filter_v_max: 255
filter_debug: true
use_field: false
field_h_min: 40
field_h_max: 334
field_s_min: 0
field_s_max: 45
field_v_min: 90
field_v_max: 253
field_debug: false
use_second_filter: false
filter2_h_min: 40
filter2_h_max: 334
filter2_s_min: 0
filter2_s_max: 45
filter2_v_min: 90
filter2_v_max: 253
ellipse_size: 3